web/public/css/_variables.less

```diff
@@ -3,12 +3,13 @@
 
 @cd-blue: #0093D5;
 @cd-light-blue: lighten(@cd-blue, 15%);;
-@cd-purple: #73449B;
-@cd-red: #9B1C20;
 @cd-green: #49B749;
 @cd-light-green: lighten(@cd-green, 15%);
 @cd-yellow: #fbcc33;
 @cd-orange: #FAA31A;
+@cd-purple: #73449B;
+@cd-light-purple: lighten(@cd-purple, 15%);
```

Comment:
@cd-light-purple: tint(@cd-purple, x%);

I'll consolidate it all to cd-commons variables over the next two days, so in the meantime, just tint it by a multiple of 10.